js/newEddie.js

```diff
@@ -0,0 +1,15 @@
+var parents = document.getElementsByTagName("SELECT");
+var attr = ["selected", "disabled", "hidden"];
+var i;
+var j;
+
+for ( i=0; i<parents.length; i++){
+    
+    children = parents[i].firstElementChild;
```

Comment:
As promised, we've started playing around with JS, so time for a quick rundown of what's happening here: since there's no `var` / `const` / `let` before `children`, it's automatically defined in the global scope. That makes little difference in your particular implementation, since everything is in the global scope by default (for now), but do keep in mind that this will need to change very soon. :wink: